src/apps/products/schemas.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from typing import Any
+from decimal import Decimal
+
+from pydantic import BaseModel, Field, validator
+
+from src.apps.products.models import Category, Product
+
+
+class CategoryBaseSchema(BaseModel):
+    name: str = Field(max_length=75)
+
+
+class CategoryInputSchema(CategoryBaseSchema):
+    pass
+
+
+class CategoryOutputSchema(CategoryBaseSchema):
+    id: int
+
+    class Config:
+        orm_model = Category
+        orm_mode = True
+
+
+class ProductBaseSchema(BaseModel):
+    name: str = Field(max_length=75)
+    price: Decimal = Field()
+    categories: list[CategoryOutputSchema] = []
```

Comment:
Move to output

Reply:
@jakub-figat  i think `categories` can't be moved to `ProductOutputSchema`, because  the data from `categories` is used in `create_product` service